packages/design-system/src/Icon/Icon.provider.tsx

```diff
@@ -986,15 +587,13 @@ export function IconProvider(props: {
 }) {
   const { color, iconName, size } = props;
 
-  const pascalCaseIconName = iconName
-    .split("-")
-    .map((word) => word[0].toUpperCase() + word.slice(1))
-    .join("");
+  // const pascalCaseIconName = iconName
+  //   .split("-")
+  //   .map((word) => word[0].toUpperCase() + word.slice(1))
+  //   .join("");
 
   const Icon = ICON_LOOKUP[iconName as keyof typeof ICON_LOOKUP]
     ? ICON_LOOKUP[iconName as keyof typeof ICON_LOOKUP]
```

Comment:
Does this mean we can use IconName as a type again? If yes, we should update all the places we are using icon - button, link, select, etc to accept an IconName and not a String.

Reply:
We will do this after this hustle... I will add TODO here